core/server/src/test/java/alluxio/worker/file/UnderFileSystemManagerTest.java

```diff
@@ -0,0 +1,87 @@
+import org.mockito.BDDMockito;
+import org.mockito.Mockito;
+import org.powermock.api.mockito.PowerMockito;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.junit4.PowerMockRunner;
+
+import java.io.OutputStream;
+
+@RunWith(PowerMockRunner.class)
+@PrepareForTest({UnderFileSystem.class})
+public final class UnderFileSystemManagerTest {
+  /**
+   * Tests creating a file with the manager will call {@link UnderFileSystem#create}.
+   */
+  @Test
+  public void createUfsFileTest() throws Exception {
+    UnderFileSystem mockUfs = Mockito.mock(UnderFileSystem.class);
+    OutputStream mockOutputStream = Mockito.mock(OutputStream.class);
+    Mockito.when(mockUfs.create(Mockito.anyString())).thenReturn(mockOutputStream);
+    PowerMockito.mockStatic(UnderFileSystem.class);
+    BDDMockito.given(UnderFileSystem.get(Mockito.anyString(), Mockito.any(Configuration.class)))
+        .willReturn(mockUfs);
+
+    String testFile = "test";
+    UnderFileSystemManager manager = new UnderFileSystemManager();
+    manager.createFile(testFile);
+    Mockito.verify(mockUfs).create(Mockito.contains(testFile));
+  }
+
+  /**
+   * Tests completing a file with the manager will call {@link UnderFileSystem#rename}.
+   */
+  @Test
+  public void completeUfsFileTest() throws Exception {
+    UnderFileSystem mockUfs = Mockito.mock(UnderFileSystem.class);
+    OutputStream mockOutputStream = Mockito.mock(OutputStream.class);
+    Mockito.when(mockUfs.create(Mockito.anyString())).thenReturn(mockOutputStream);
+    PowerMockito.mockStatic(UnderFileSystem.class);
+    BDDMockito.given(UnderFileSystem.get(Mockito.anyString(), Mockito.any(Configuration.class)))
+        .willReturn(mockUfs);
+
+    String testFile = "test";
+    UnderFileSystemManager manager = new UnderFileSystemManager();
+    long id = manager.createFile(testFile);
+    Mockito.verify(mockUfs).create(Mockito.contains(testFile));
+    manager.completeFile(id);
+    Mockito.verify(mockUfs).rename(Mockito.contains(testFile), Mockito.eq(testFile));
+  }
+
+  /**
+   * Tests canceling a file with the manager will call {@link UnderFileSystem#delete}.
+   */
+  @Test
+  public void cancelUfsFileTest() throws Exception {
+    UnderFileSystem mockUfs = Mockito.mock(UnderFileSystem.class);
+    OutputStream mockOutputStream = Mockito.mock(OutputStream.class);
+    Mockito.when(mockUfs.create(Mockito.anyString())).thenReturn(mockOutputStream);
+    PowerMockito.mockStatic(UnderFileSystem.class);
+    BDDMockito.given(UnderFileSystem.get(Mockito.anyString(), Mockito.any(Configuration.class)))
+        .willReturn(mockUfs);
+
+    String testFile = "test";
+    UnderFileSystemManager manager = new UnderFileSystemManager();
+    long id = manager.createFile(testFile);
+    Mockito.verify(mockUfs).create(Mockito.contains(testFile));
+    manager.cancelFile(id);
+    Mockito.verify(mockUfs).delete(Mockito.contains(testFile), Mockito.eq(false));
+  }
```

Comment:
Can we add cases to test exception thrown cases (`FileDoesNotExistException`) for the above three operations?